themes/custom/droopler_theme/templates/layout/page.html.twig

```diff
@@ -138,16 +138,22 @@
       <div class="{{ container }}">
         {{ page.footer_main }}
         {% if page.footer_first or page.footer_second or page.footer_third or page.footer_fourth %}
-          <div class="site-footer__top clearfix">
-            {{ page.footer_first }}
-            {{ page.footer_second }}
-            {{ page.footer_third }}
-            {{ page.footer_fourth }}
-          </div>
-        {% endif %}
-        {% if page.footer_fifth %}
-          <div class="site-footer__bottom">
-            {{ page.footer_fifth }}
+          <div class="site-footer-blocks row">
+            {% for i, region in filled_regions %}
+              {% if filled_regions|length == i + 1  %}
+                {% if (i + 1) % 2 == 1 %}
+                  {% set classes = 'col' %}
+                {% else %}
+                  {% set classes = ['col-6', 'col-xl'] %}
+                {% endif %}
+              {% else %}
+                {% set classes = ['col-6', 'col-xl-2'] %}
+              {% endif %}
+
+              <div{{ attributes.addClass(classes) }}>
```

Comment:
Probably will be better to create own attributes rather use general page attributes for the specific region.
{% set footer_attributes = create_attribute() %}
<div{{ footer_attributes.addClass(classes) }}>